src/SprykerEco/Shared/Computop/ComputopConstants.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Shared\Computop;
+
+interface ComputopConstants
+{
+
+    const PROVIDER_NAME = 'Computop';
+    const PAYMENT_METHOD_CREDIT_CARD = 'ComputopCreditCard';
+    const PAYMENT_METHOD_PAY_PAL = 'ComputopPayPal';
+    const PAYMENT_METHOD_DIRECT_DEBIT = 'ComputopDirectDebit';
+    const PAYMENT_METHOD_SOFORT = 'ComputopSofort';
+
+    //Statuses
+    const COMPUTOP_OMS_STATUS_NEW = 'new';
+    const COMPUTOP_OMS_STATUS_AUTHORIZED = 'authorized';
```

Comment:
Where are those used? I am pretty sure all the constants here violate the usage principle.
These here are only for use in env config, nothing else.
I know the naming is really bad - this leads to confusion.
We must put these into the module Config etc instead. Or add a custom constants file unrelated to the env config ones.

Reply:
Mark is right, please move statuses to Zed module config if they are necessary only for Zed.

Reply:
What should contain those configs here would probably be a Config file in Shared:

    src/SprykerEco/Shared/Computop/ComputopConfig.php